cmd/nodecmd/sync.go

```diff
@@ -38,6 +41,65 @@ You can check the subnet bootstrap status by calling avalanche node status <clus
 	return cmd
 }
 
+func getNodesWoEIPInAnsibleInventory(clusterNodes []string) []string {
+	nodesWoEIP := []string{}
+	for _, node := range clusterNodes {
+		nodeConfig, err := app.LoadClusterNodeConfig(node)
+		if err != nil {
+			continue
+		}
+		if nodeConfig.ElasticIP == "" {
+			nodesWoEIP = append(nodesWoEIP, node)
+		}
+	}
+	return nodesWoEIP
+}
+
+func getPublicIPForNodesWoEIP(nodesWoEIP []string) (map[string]string, error) {
+	lastRegion := ""
+	var ec2Svc *ec2.EC2
+	publicIPMap := make(map[string]string)
+	for _, node := range nodesWoEIP {
+		nodeConfig, err := app.LoadClusterNodeConfig(node)
+		if err != nil {
+			return nil, err
+		}
+		if nodeConfig.Region != lastRegion {
+			sess, err := getAWSCloudCredentials(nodeConfig.Region, constants.GetAWSNodeIP)
+			if err != nil {
+				return nil, err
+			}
+			ec2Svc = ec2.New(sess)
+			lastRegion = nodeConfig.Region
+		}
+		publicIP, err := awsAPI.GetInstancePublicIPs(ec2Svc, []string{node})
+		if err != nil {
+			return nil, err
+		}
+		publicIPMap[node] = publicIP[node]
+	}
+	return publicIPMap, nil
+}
+
+func handleNodesWoEIP(clusterName string) error {
```

Comment:
probably give a better name, like updateAnsiblePublicIPs

Reply:
I believe you can just get the IPs for all the nodes in the cluster, disregarding if they have or not elastic IP

Reply:
probably we want for node list to also list the updated node IPs for the user to so ssh or whatever (maybe also give ssh instructions) in an extended output flag for node list